test/client/fixtures/sandbox/code-instrumentation/location-test.js

```diff
@@ -157,6 +192,83 @@ test('special pages (GH-339)', function () {
     destLocation.forceLocation(storedForcedLocation);
 });
 
+test('should add the trailing slash to location.href using href.set(), assign() and replace() functions if url consist of origin (GH-1426)', function () {
```

Comment:
`should ensure a trailing slash on page navigation using href setter, assign and replace methods (GH-1426)`